src/test/groovy/pl/allegro/tech/build/axion/release/domain/VersionResolverTest.groovy

```diff
@@ -20,6 +20,24 @@ class VersionResolverTest extends RepositoryBasedTest {
 
     VersionProperties defaultVersionRules = versionProperties().build()
 
+    @Shared
```

Comment:
i would rather have those created in test, close to the usage context instead of class level. also they only differ in `prefix`, so you can add prefix as parameter of newly created test and all will be much more readable :)

Reply:
You're quite right, I have revised the test.